app/api/customer/models.py

```diff
@@ -2,15 +2,20 @@
 
 from sqlmodel import Field, SQLModel
 
+class CustomerUpdate(SQLModel):
+    name: str | None = None
+    email: str | None = None
+    phone_number: str | None = None
+    is_ower: bool | None = None
 
-class CustomerCreateUpdate(SQLModel):
+class CustomerCreate(SQLModel):
     name: str = Field()
     email: str = Field()
     phone_number: str = Field()
```

Comment:
Let's add here (in this PR as well) uniqoe constraints for this fields (`email` and `phone_number`)
As this changes lie on db level you should also create the migration for that